.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+##### Thanks for considering contributing to HTTPX!
+
+Our [documentation on contributing to HTTPX]() contains information on
```

Comment:
Missing link? 😄 Also a wording suggestion with a clearer call to action towards the docs.

```suggestion
To get started, please see our [documentation on contributing to HTTPX](https://www.encode.io/httpx/contributing/). It contains information on
```